ee/candi/cloud-providers/openstack/bashible/bundles/common-steps/bootstrap-networks.sh.tpl

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+{{- /*
+# Copyright 2024 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/}}
+shopt -s extglob
+
+if ! which netplan 2>/dev/null 1>&2; then
+  exit 0
+fi
+
+function cat_file() {
+  cat_dev=$1
+  cat_metric=$2
+  cat_mac=$3
+  cat > /etc/netplan/100-cim-"$cat_dev".yaml <<BOOTSTRAP_NETWORK_EOF
+network:
+  version: 2
+  ethernets:
+    $cat_dev:
+      dhcp4-overrides:
+        route-metric: $cat_metric
+      match:
+        macaddress: $cat_mac
+BOOTSTRAP_NETWORK_EOF
+}
+
+count_default=$(ip route show default | wc -l)
+if [[ "$count_default" -gt "1" ]]; then
+  configured_macs="$(grep -Po '(?<=macaddress: ).+' /etc/netplan/50-cloud-init.yaml)"
+  for mac in $configured_macs; do
+    ifname="$(ip -o link show | grep "link/ether $mac" | cut -d ":" -f2 | tr -d " ")"
```

Comment:
i would prefer ip -j and jq, but i dont know if there is jq in system.

Reply:
At the moment of script execution our jq is not in the system yet, so I can't use it.